### PR TITLE
LPS-73482 adding middle mouse button functionality to site selector

### PR DESCRIPTION
**Pull request by balazssk**
Hi Zsaga,

The current fix only tackles the middle mouse functionality on links. This means that both descriptive and list displayStyle should be fine.
The vertical card however can't be encompassed by a simple <a>tag, as there is a possibility for it to have a footer linking to child sites.
Can you please have a look at it?

Thanks,
Balazs

### CODE REVIEW

**Comment by zsagia** [comment]
Hi Balázs,

In that case if we want to provide a similar functionality as we had in our previous version we should implement the same behavior for icon, descriptive and list views as well when we use middle mouse button or CTRL + left click. 
After I checked your solution I found the modified behavior is inconsistent a bit. I think this inconsistency comes from the original solution that always register an event handler for click event on every 'selector-button' element.
I'm sure developers won't approve this half ready solution and will ask us to finalize it.
Basically, this kind of behavior can be find in 48 places in portal-master code and I don't think we need to start modify them.

Anyway, thanks a lot for your pull but I have to close it.

**Comment by zsagia** [comment]
Forwarded to Dustin: https://github.com/dustinryerson/liferay-portal-ee/pull/3034.

Thanks
